### PR TITLE
[Imaging Browser] Prevented header-info dropdown from going out of alignment

### PR DESCRIPTION
**Pull request by AnyhowStep**
In reference to Redmine ticket 12228, the "header info" part currently goes out of the "header" with very long strings or when the screen width is too small.

This pull request fixes it by adding some styles that keep the text in check.

### CODE REVIEW

**Comment by xlecours** [comment]
This is not my cup of tea.

**Comment by johnsaigle** [comment]
I don't know if it's good practice to start mixing in style changes into the .js files. Is there no way to correct this via CSS?

**Comment by AnyhowStep** [comment]
I'm not a front-end guy =(
@jacobpenny , what's the recommended way for me to add this change?

**Comment by jacobpenny** [comment]
Instead of using inline styles you could make the changes here: https://github.com/AnyhowStep/Loris/blob/900a9211e801b6e6705d8335e6321c443ed270e6/modules/imaging_browser/css/imaging_browser.css#L72

I think that would be best in this case.

**Comment by AnyhowStep** [comment]
Gdi, bootstrap.

**Comment by johnsaigle** [comment]
still not working :(

**Comment by AnyhowStep** [comment]
Fuck

On Chrome
image

On Firefox
image

**Comment by AnyhowStep** [comment]
Cleared cache on Firefox. Seems to work now.
image

File: modules/imaging_browser/jsx/ImagePanel.js [hunk 1]
```diff
@@ -39,13 +39,13 @@ var ImagePanelHeader = React.createClass({
...
-                </div>
-            </div>
+                </span>
+            </span>
             );
     return (
 
-            <div className="panel-heading">
-            <input type="checkbox" data-file-id={this.props.FileID}
+            <div className="panel-heading clearfix">
```
Comment (driusan) [inline modules/imaging_browser/jsx/ImagePanel.js]: is the clearfix really necessary? I don't think we need to support IE6.

Comment (johnsaigle) [inline modules/imaging_browser/jsx/ImagePanel.js]: In fact officially we only support Firefox and Chrome!

Comment (AnyhowStep) [inline modules/imaging_browser/jsx/ImagePanel.js]: It is necessary. Without it, it breaks on FF and Chrome. I don't know why. I am no front-end guy. I just Googled and it worked.

Comment (AnyhowStep) [inline modules/imaging_browser/jsx/ImagePanel.js]: I didn't even bother to look at what styles the clearfix class has, lol

Comment (driusan) [inline modules/imaging_browser/jsx/ImagePanel.js]: I've never heard of a CSS "clearfix" being anything other than an IE6 fix, but okay.. if it's necessary it's fine. (Though this has conflicts again.)

**Comment by MounaSafiHarab** [comment]
@johnsaigle 

added the rebase because of the conflicts; so removed the passed manual test so we can re-test!

**Comment by johnsaigle** [comment]
@MounaSafiHarab To be clear does this need rebasing or only testing?

**Comment by MounaSafiHarab** [comment]
@johnsaigle 

There is a conflict, hence the Needs rebase tag. But once rebased, I assume we need to re-tests, non?
so I would say both..

**Comment by johnsaigle** [comment]
Conflicts have been resolved

**Comment by driusan** [comment]
This needs to be rebased, not have a merge commit added. The merge commit also needs to be removed now..

**Comment by johnsaigle** [comment]
Rebased, should be ready for testing.

**Comment by johnsaigle** [comment]
The build succeeds on PHP 7.1 but not on 7.0.

It says `npm install` didn't work?

Not sure what to do about this 😕

**Comment by kongtiaowang** [comment]
@jacobpenny  It works

**Comment by MounaSafiHarab** [comment]
@johnsaigle 

I did test it and it works fine.
One minor comment; in brainbrowser, when the name is too long and we replace it with `...`, we usually have a hover over option to display the full name so we can see it so this would have been really useful to have here as well; but maybe not related to this pull request.

titleresizes

**Comment by johnsaigle** [comment]
@MounaSafiHarab Would you mind making a 17.2 Redmine ticket to address this?

Thanks for the review!

**Comment by MounaSafiHarab** [comment]
@johnsaigle 

done; redmine 12786!

**Pull request merged**